### PR TITLE
Fixes #2: Added PR template

### LINKED ISSUE #2: No PR template

### PR DESCRIPTION
**Pull request by ayush0x00**
Fixes #2

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Also list any relevant details for your test configuration
+
+- [ ] Test A
+- [ ] Test B
+
+**Test Configuration**:
```
Comment (danishjamal104) [inline .github/pull_request_template.md]: This repo is only for storing lambda files, hence no test configuration is required please remove this.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* Hardware:
+* Additional configurations:
+
+## Checklist:
+
+- [ ] I have performed a self-review of my own code
+- [ ] I have commented my code, particularly in hard-to-understand areas
+- [ ] I have made corresponding changes to the documentation
+- [ ] My changes generate no new warnings
```
Comment (danishjamal104) [inline .github/pull_request_template.md:20]: No test cases hence remove this.

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- [ ] New and existing unit tests pass locally with my changes
```
Comment (danishjamal104) [inline .github/pull_request_template.md]: remove this too, same as previous one

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
+# Description
+
+Include a summary of the change and which issue is fixed. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
```
Comment (danishjamal104) [inline .github/pull_request_template.md:7]: Only possibility changes are update in previous lambdas or create a new one so remove the default checklist and add this
```md
- [ ] Update lambda @<name of lambda>
- [ ] Created new lambda @<name of lambda>
- [ ] Other 
```

**Review by danishjamal104 — CHANGES_REQUESTED** [review]
Also, you haven't mentioned anything regarding the title of the PR. Please address that also. Thanks.

**Comment by danishjamal104** [comment]
@ayush0x00 , you are making changes directly to the master. Not preferable at all.

**Review by danishjamal104 — APPROVED** [review]
LGTM

**Comment by danishjamal104** [comment]
@ayush0x00 , edit the title of PR to follow the convention.

**Review by chhavituteja — APPROVED** [review]

**Review by chhavituteja — APPROVED** [review]

**Pull request merged**